vmtkScripts/vmtkscripts.py

```diff
@@ -1,142 +1,144 @@
-  'vmtkmeshtosurface',
-  'vmtkmeshtransform',
-  'vmtkmeshtransformtoras',
-  'vmtkmeshvectorfromcomponents',
-  'vmtkmeshviewer',
-  'vmtkmeshvolume',
-  'vmtkmeshvorticityhelicity',
-  'vmtkmeshwallshearrate',
-  'vmtkmeshwriter',
-  'vmtknetworkeditor',
-  'vmtknetworkextraction',
-  'vmtknetworkwriter',
-  'vmtkparticletracer',
-  'vmtkpathlineanimator',
-  'vmtkpointsplitextractor',
-  'vmtkpointtransform',
-  'vmtkpolyballmodeller',
-  'vmtkpotentialfit',
-  'vmtkpythonscript',
-  'vmtkrenderer',
-  'vmtkrendertoimage',
-  'vmtkrbfinterpolation',
-  'vmtksurfaceappend',
-  'vmtksurfacearraysmoothing',
-  'vmtksurfacearrayoperation',
-  'vmtksurfacebooleanoperation',
-  'vmtksurfacecapper',
-  'vmtksurfacecelldatatopointdata',
-  'vmtksurfacecenterlineprojection',
-  'vmtksurfaceclipper',
-  'vmtksurfacecliploop',
-  'vmtksurfaceconnectivity',
-  'vmtksurfacecurvature',
-  'vmtksurfacedecimation',
-  'vmtksurfacedistance',
-  'vmtksurfaceendclipper',
-  'vmtksurfacekiteremoval',
-  'vmtksurfaceloopextraction',
-  'vmtksurfacemassproperties',
-  'vmtksurfacemodeller',
-  'vmtksurfacenormals',
-  'vmtksurfacepointdatatocelldata',
-  'vmtksurfacepolyballevaluation',
-  'vmtksurfaceprojection',
-  'vmtksurfacereader',
-  'vmtksurfacereferencesystemtransform',
-  'vmtksurfaceregiondrawing',
-  'vmtksurfaceremeshing',
-  'vmtksurfacescaling',
-  'vmtksurfacesmoothing',
-  'vmtksurfacesubdivision',
-  'vmtksurfacetransform',
-  'vmtksurfacetransforminteractive',
-  'vmtksurfacetransformtoras',
-  'vmtksurfacetriangle',
-  'vmtksurfacetomesh',
-  'vmtksurfaceviewer',
-  'vmtksurfacewriter',
-  'vmtksurfmesh',
-  'vmtktetgen',
-  'vmtktetringenerator'
-  ]
+    'vmtk.vmtkactivetubes',
+    'vmtk.vmtkbifurcationprofiles',
+    'vmtk.vmtkbifurcationreferencesystems',
+    'vmtk.vmtkbifurcationsections',
+    'vmtk.vmtkbifurcationvectors',
+    'vmtk.vmtkboundarylayer',
+    'vmtk.vmtkboundaryreferencesystems',
+    'vmtk.vmtkbranchclipper',
+    'vmtk.vmtkbranchextractor',
+    'vmtk.vmtkbranchgeometry',
+    'vmtk.vmtkbranchmapping',
+    'vmtk.vmtkbranchmetrics',
+    'vmtk.vmtkbranchpatching',
+    'vmtk.vmtkbranchsections',
+    'vmtk.vmtkcenterlineattributes',
+    'vmtk.vmtkcenterlinegeometry',
+    'vmtk.vmtkcenterlineinterpolation',
+    'vmtk.vmtkcenterlinelabeler',
+    'vmtk.vmtkcenterlinemerge',
+    'vmtk.vmtkcenterlinemodeller',
+    'vmtk.vmtkcenterlineoffsetattributes',
+    'vmtk.vmtkcenterlineresampling',
+    'vmtk.vmtkcenterlines',
+    'vmtk.vmtkcenterlinesections',
+    'vmtk.vmtkcenterlinesmoothing',
+    'vmtk.vmtkcenterlineviewer',
+    'vmtk.vmtkdelaunayvoronoi',
+    'vmtk.vmtkdistancetocenterlines',
+    'vmtk.vmtkendpointextractor',
+    'vmtk.vmtkflowextensions',
+    'vmtk.vmtkicpregistration',
+    'vmtk.vmtkimagebinarize',
+    'vmtk.vmtkimagecast',
+    'vmtk.vmtkimagecompose',
+    'vmtk.vmtkimagecurvedmpr',
+    'vmtk.vmtkimagefeaturecorrection',
+    'vmtk.vmtkimagefeatures',
+    #'vmtk.vmtkimageinitialization',
```

Comment:
Hey luca, thanks for catching this! I thought I had taken care of it. 

There was an issue for a while with circular imports when running the scripts under Python 2. The six scripts that I had commented out each had an `from vmtk import vmtkscripts` statement at their header which was trying to execute when being imported in the `vmtkscripts` file - not great. I just rearranged the location of the import statement in these six sicripts, and everything seems to work well now for both python2 and 3.  Just pushed the change up now.